apps/voting/app/src/components/VoteActions.js

```diff
@@ -254,10 +254,9 @@ const TokenReference = ({
   startDate,
 }) => (
   <Info>
-    Voting with <span css="font-weight: bold;">{userBalance}</span> of your{' '}
-    <span css="font-weight: bold;">{tokenSymbol}</span> at block{' '}
-    <span css="font-weight: bold;">{snapshotBlock}</span> due to the vote
-    starting at <span css="font-weight: bold;">{formatDate(startDate)}</span>.
+    Voting with <strong>{userBalance}</strong> of your{' '}
+    <strong>{tokenSymbol}</strong> at block <strong>{snapshotBlock}</strong> due
+    to the vote starting at <strong>{formatDate(startDate)}</strong>.
```

Comment:
I'm really pulling at hairs here with what I consider semantic html, but to clarify on the `<strong>` vs. css styling; in cases where the bold is **not** really meant for emphasis, it's fine to use css. I don't see the user balance, token symbol, and snapshot block as being emphasized; you wouldn't stress them when reading them out loud (unlike, for example, a stress on "anyone").

Anyhow, there's no need to change anything but perhaps we'll all learn something from a discussion (again, this is what I've considered semantic and I may well be wrong) :).

Reply:
Yeah, I don't disagree but then is it really ok to add visual effects that have nothing to do with the interpretation of the text but use the same style of other effects that do give the text semantic meaning? Anyhow, let's figure this one out and update as needed.

Reply:
Yeah this is true, especially when they're inline with other text. Perhaps if they were on different headings or etc. it would be more appropriate to use just a css style.